.rubocop.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
   - "vendor/**/*"
   - "db/**/*"
   - "client/**/*"
+  - "app/jobs/simulate_bus_location_job.rb"
```

Comment:
The calculation for the 'new_latitude' and 'new_longitude' kept triggering an error "Assignment Branch Condition size for perform is too high". I tried refactoring but it was becoming more and more complicated.